apps/generic_browser.talon

```diff
@@ -1,7 +1,8 @@
 tag: browser
 -
-(address bar | go address | go url): browser.focus_address()
-(address copy | url copy | copy address | copy url):
+address bar | go address | go url: browser.focus_address()
+duplicate tab | tab duplicate: user.duplicate_tab()
```

Comment:
I'd move this to misc/tabs.talon instead since the action doesn't apply to just browsers. I think that duplication is probably about as common as the 'tab reopen' functionality for example which is in the misc/tabs.talon file. VSCode for example also has the ability to duplicate tabs.

Reply:
Yeah good call, legacy from how I built it for the browser tag to begin with, my bad.